src/gui/static/src/app/components/pages/exchange/exchange-create/exchange-create.component.ts

```diff
@@ -45,25 +48,30 @@ export class ExchangeCreateComponent implements OnInit, OnDestroy {
   amountTooLow = false;
   amountTooHight = false;
 
+  // If the user has acepted the agreement.
   private agreement = false;
+
   private subscriptionsGroup: SubscriptionLike[] = [];
   private exchangeSubscription: SubscriptionLike;
   private priceUpdateSubscription: SubscriptionLike;
 
-  get toAmount() {
+  // Approximately how many coins will be received for the amount of coins the user will send,
+  // as per the value entered on the form and the current price.
+  get toAmount(): string {
     if (!this.activeTradingPair) {
-      return 0;
+      return '0';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/skycoin/skycoin/pullRequest?prid=7163600)
```suggestion
      return "0";
```